<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsNewConnectionBase</class>
 <widget class="QDialog" name="QgsNewConnectionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create a New PostGIS connection</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>87</width>
         <height>150</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="Q3GroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Connection Information</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel1_2" >
            <property name="text" >
             <string>Name</string>
            </property>
            <property name="buddy" >
             <cstring>txtName</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel1" >
            <property name="text" >
             <string>Host</string>
            </property>
            <property name="buddy" >
             <cstring>txtHost</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2" >
            <property name="text" >
             <string>Database</string>
            </property>
            <property name="buddy" >
             <cstring>txtDatabase</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2_2" >
            <property name="text" >
             <string>Port</string>
            </property>
            <property name="buddy" >
             <cstring>txtPort</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3" >
            <property name="text" >
             <string>Username</string>
            </property>
            <property name="buddy" >
             <cstring>txtUsername</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3_2" >
            <property name="text" >
             <string>Password</string>
            </property>
            <property name="buddy" >
             <cstring>txtPassword</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txtName" >
            <property name="toolTip" >
             <string>Name of the new connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtHost" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtDatabase" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtPort" >
            <property name="text" >
             <string>5432</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUsername" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkStorePassword" >
          <property name="text" >
           <string>Save Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect" >
          <property name="text" >
           <string>Test Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_geometryColumnsOnly" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;">Restrict the displayed tables to those that are in the geometry_columns table&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;">Restricts the displayed tables to those that are in the geometry_columns table. This can speed up the initial display of spatial tables.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Only look in the geometry_columns table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_publicSchemaOnly" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:12pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Restrict the search to the public schema for spatial tables not in the geometry_columns table&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:12pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">When searching for spatial tables that are not in the geometry_columns tables, restrict the search to tables that are in the public schema (for some databases this can save lots of time)&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Only look in the 'public' schema</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
